via/services/youtube.py

```diff
@@ -0,0 +1,23 @@
+from urllib.parse import parse_qs, urlparse
+
+
+class YoutubeService:
+    @classmethod
+    def parse_url(cls, public_url):
+        """Get the youtube video ID from an URL."""
+        parsed = urlparse(public_url)
```

Comment:
Once we integrate this into LMS we might have to extend this to support our own `video://youtube/ID` type of URL if we go that route.

For now this allows for easier testing directly in via.